### ISSUE TITLE
[convnet] Scale by alpha layer in convnet diagram

### ISSUE THREAD

**Issue by dstepanenko** [prog-eval/prog-eval#1]
Hello, I was implementing code according to convnet diagram and figured out that I have no idea, what does lat but one layer of it means. It's name is "Scale by alpha = 0.125". Could you please help me?
https://github.com/prog-eval/prog-eval/blob/master/convnet/docs/convnet_diagram.png

**Comment by prog-eval** [comment]
This indicates that you should scale the logits of the last layer by 0.125 before applying the softmax activation.

This is a trick to improve training stability.

**Comment by dstepanenko** [comment]
> scale the logits

so, what layer should I use there in terms of keras layers?

**Comment by prog-eval** [comment]
I can't speak to how it's done in specific libraries, but mathematically it looks like
```
...
x = global_maxpool(x)  # global maxpool
x = x @ W_linear       # linear layer, where `@` is matrix multiplication and W_linear is a 128x2 weight matrix
x = x * 0.125          # scale logits
x = softmax(x)         # apply softmax
loss = categorical_cross_entropy(x, targets)
```
